precache/package-lock.json

```diff
@@ -1,5634 +1,8 @@
 {
   "name": "precache",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
น่าสนใจ... ฉันไม่ได้คาดหวังสิ่งนี้! ฉันเดาว่าเรามี NPM เวอร์ชันต่างๆ
Interesting... I wasn't expecting this! I guess we have different versions of NPM.

Reply:
I'm worried that I might be running the wrong command.

Reply:
This file is automatically generated. I don't think you could have made an error.

If you're curious you could try running `npm -v`. Mine looks like this:
```
❯ npm -v
7.16.0
```